README.md

```diff
@@ -500,6 +508,54 @@ end
 
 Note that this approach may cause unexpected behavior if your backend sends the Referer HTTP header (which is unlikely).
 
+## SSL usage
+
+Unfortunately we cannot setup the runtime certificate authority on your browser
+at time of configuring the Capybara driver.  So you need to take care of this
+step yourself as a prepartion. A good point would be directly after configuring
+this gem.
+
+### Google Chrome Headless example
+
+Google Chrome/Chromium is capable to run as a test browser with the new
+headless mode which is not able to handle the deprecated
+`--ignore-certificate-errors` flag. But the headless mode is capable of
+handling the user PKI certificate store.  So you just need to import the
+runtime Puffing Billy certificate authority on your system store, or generate a
+new store for your current session. The following examples demonstrates the
+former variant:
+
+```ruby
+# Overwrite the local home directory for chrome. We use this
+# to setup a custom SSL certificate store.
+ENV['HOME'] = "#{Dir.tmpdir}/chrome-home-#{Time.now.to_i}"
+
+# Clear and recreate the Chrome home directory.
+FileUtils.rm_rf(ENV['HOME'])
```

Comment:
Using `Time.now` above should pretty much guarantee this directory will never exist right?

Reply:
We can do the `Time.now` trick, by then we have a mess while cleaning up. I think of a new configured directory like `cache_dir` and `certs_dir`. I also do not like the manipulation of the `HOME` environment variable. But it's just for child processes. It would be better to have a handling like written below.